components/table/table-wrapper.js

```diff
@@ -249,7 +249,13 @@ export const tableStyles = css`
 
 	/* sticky + scroll-wrapper */
 	d2l-table-wrapper[sticky-headers][sticky-headers-scroll-wrapper] .d2l-table {
-		display: block;
+		display: flex;
```

Comment:
Is this forcing it to 100% width where it wasn't before?

Reply:
And if there's a background color you get this:

![Screenshot 2025-06-20 at 11 38 40 AM](https://github.com/user-attachments/assets/529f85df-7797-4bee-ad4c-5193793bdc54)